presto-main/src/main/java/com/facebook/presto/operator/scalar/MapTransformKeyFunction.java

```diff
@@ -189,7 +190,8 @@ private static MethodHandle generateTransformKey(Type keyType, Type transformedK
         body.append(typedSet.set(newInstance(
                 TypedSet.class,
                 constantType(binder, transformedKeyType),
-                divide(positionCount, constantInt(2)))));
+                divide(positionCount, constantInt(2)),
+                BytecodeExpressions.constantString(MAP_TRANSFORM_KEY_FUNCTION.getSignature().getName()))));
```

Comment:
yea i saw this pattern, but keep forgetting to hit option-enter to import 

hoping there's an auto-import option in idea